samples/gitchurn/src/gitChurnMain/kotlin/GitChurn.kt

```diff
@@ -35,33 +35,44 @@ fun main(args: Array<String>) {
 private fun calculateChurn(workDir: String, limit: Int) {
     println("Opening…")
     val repository = git.repository(workDir)
-    val map = mutableMapOf<String, Int>()
+    data class WhoAndWhere(val authorEmail: String = "", val filePath: String = "")
+    val modificationsByAuthor = mutableMapOf<WhoAndWhere, Int>()
     var count = 0
-    val commits = repository.commits()
-    val limited = commits.take(limit)
+    val commits = repository.myCommits()
+    val commitsLimit = commits.take(limit)
```

Comment:
`commitsLimit` looks misnamed. Also `commits` is used only once, so in Kotlin this code should be written more like
```
val commits = repository.myCommits().take(limit)
```